packages/lib-user/src/components/GroupStats/GroupStatsContainer.js

```diff
@@ -1,32 +1,39 @@
 'use client'
 
+import asyncStates from '@zooniverse/async-states'
+import { Notification } from 'grommet'
 import { object, string } from 'prop-types'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 
 import {
   usePanoptesAuthUser,
-  usePanoptesProjects,
   usePanoptesUserGroup,
-  useStats
 } from '@hooks'
 
 import {
+  createPanoptesMembership,
   deletePanoptesUserGroup,
   getBearerToken,
-  getDateInterval,
   updatePanoptesUserGroup
 } from '@utils'
 
+import { getStatus } from './helpers/getStatus'
```

Comment:
I don't love the name of this helper function, open to suggestion.

Reply:
Would `getJoinStatus` or `getUserGroupStatus` make sense?

Reply:
I like `getUserGroupStatus` better than just `getStatus` 👍